tensilelite/Tensile/CustomKernels/Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941.s

```diff
@@ -0,0 +1,9297 @@
+
+/******************************************/
+/* Begin Kernel                           */
+/******************************************/
+.amdgcn_target "amdgcn-amd-amdhsa--gfx941"
+.text
+.protected Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941
+.globl Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941
+.p2align 8
+.type Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941,@function
+.section .rodata,#alloc
+.p2align 6
+.amdhsa_kernel Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941
+  .amdhsa_user_sgpr_kernarg_segment_ptr 1
+  .amdhsa_accum_offset 128 // accvgpr offset
+  .amdhsa_next_free_vgpr 256 // vgprs
+  .amdhsa_next_free_sgpr 84 // sgprs
+  .amdhsa_group_segment_fixed_size 32768 // lds bytes
+  .amdhsa_private_segment_fixed_size 0
+  .amdhsa_system_sgpr_workgroup_id_x 1
+  .amdhsa_system_sgpr_workgroup_id_y 1
+  .amdhsa_system_sgpr_workgroup_id_z 1
+  .amdhsa_system_vgpr_workitem_id 0
+  .amdhsa_float_denorm_mode_32 3
+  .amdhsa_float_denorm_mode_16_64 3
+  .amdhsa_user_sgpr_count 13
+  .amdhsa_user_sgpr_kernarg_preload_length 11
+  .amdhsa_user_sgpr_kernarg_preload_offset 0
+.end_amdhsa_kernel
+.text
+/* Num VGPR   =112 */
+/* Num AccVGPR=144 */
+/* Num SGPR   =84 */
+
+/******************************************/
+/* Optimizations and Config:              */
+/******************************************/
+/* ThreadTile= 16 x 9 */
+/* SubGroup= 16 x 16 */
+/* VectorWidthA=4 */
+/* VectorWidthB=1 */
+/* GlobalReadVectorWidthA=4, GlobalReadVectorWidthB=2 */
+/* DirectToLdsA=False */
+/* DirectToLdsB=False */
+/* UseSgprForGRO=1 */
+.amdgpu_metadata
+---
+custom.config:
+   ProblemType:
+      OperationType: GEMM
+      DataTypeA: f8
+      DataTypeB: h
+      UseScaleAB: True
+      DataType: h
+      DestDataType: h
+      ComputeDataType: s
+      HighPrecisionAccumulate: True
+      TransposeA: False
+      TransposeB: False
+      UseBias: True
+      Activation: True
+      UseScaleAlphaVec: True
+      UseBeta: True
+      Batched: True
+      GroupedGemm:   True
+      SupportUserArgs: True
+   MatrixInstruction: [16, 16, 16, 1, 1, 4,9, 4,1]
+   1LDSBuffer: 1
+   ScheduleIterAlg: 3
+   DepthU: 32
+   StaggerU: 0
+   WorkGroupMapping: 1
+   WaveSeparateGlobalReadA: 1
+   WaveSeparateGlobalReadB: 1
+   GlobalReadVectorWidthA: 4
+   GlobalReadVectorWidthB: 2
+   AssertFree0ElementMultiple: 4
+   AssertSummationElementMultiple: 1
+   WorkGroupMappingXCC: 1
+   NoReject: 1
+amdhsa.version:
+  - 1
+  - 0
```

Comment:
" - 0" should be replaced with "- 1" for CodeObjectV4

Reply:
done